src/schema-manager.ts

```diff
@@ -151,34 +148,15 @@ export class PolygonSchema {
           txnHash: schemaTxnReceipt.hash,
           to: schemaTxnReceipt.to,
           from: schemaTxnReceipt.from,
```

Comment:
we can also remove `from` and `to` as well

Reply:
ok